### PR TITLE
Adding basic doc comments for System.Math.Sign, System.Math.SinCos, and System.MathF.SinCos

### PR DESCRIPTION
**Pull request by tannergooding**
This resolves https://github.com/dotnet/runtime/issues/54882

### CODE REVIEW

File: xml/System/Math.xml [hunk 1]
```diff
@@ -7036,10 +7036,39 @@ The following example demonstrates how to use the <xref:System.Math.Round%28Syst
...
+ </description></item></list></returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Examples  
+ The following example demonstrates how to use the <xref:System.Math.Sign%28System.IntPtr%29> method to determine the sign of an <xref:System.IntPtr> value and display it to the console.  
+
+ :::code language="cpp" source="~/samples/snippets/cpp/VS_Snippets_CLR/math.sign/CPP/sign.cpp" id="Snippet1":::
+ :::code language="csharp" source="~/samples/snippets/csharp/VS_Snippets_CLR/math.sign/CS/sign.cs" interactive="try-dotnet" id="Snippet1":::
+ :::code language="vb" source="~/samples/snippets/visualbasic/VS_Snippets_CLR/math.sign/VB/sign.vb" id="Snippet1":::
```
Comment (carlossanlop) [inline xml/System/Math.xml:7068]: These examples do not show how to use `System.Math.Sign` with `System.IntPtr`, as the above statement implies. Either the examples need to be updated, or this `## Examples` section removed.

Same applies to the examples in the other two APIs.

Comment (tannergooding) [reply xml/System/Math.xml:7068]: Where are the samples and what's the correct way to update them?

Comment (tannergooding) [reply xml/System/Math.xml:7068]: Nevermind, I found them.

Comment (carlossanlop) [reply xml/System/Math.xml:7068]: In the samples folder:
https://github.com/dotnet/dotnet-api-docs/blob/main/samples/snippets/cpp/VS_Snippets_CLR/math.sign/CPP/sign.cpp

**Review by carlossanlop — APPROVED** [review]

**Comment by carlossanlop** [comment]
@gewarren first time I see the CI failure "Snippets 5000 / build (pull_request)". The error is:

```
Error: 😵 Project missing. A project (and optionally a solution file) must be in this directory or one of the parent directories to validate and build this code.
Error: 😵 Project missing. A project (and optionally a solution file) must be in this directory or one of the parent directories to validate and build this code.
Error: Process completed with exit code 1.
```

The example files already existed before. Should we do something about this or can we ignore it?

**Comment by gewarren** [comment]
> @gewarren first time I see the CI failure "Snippets 5000 / build (pull_request)". The error is:
> 
> ```
> Error: 😵 Project missing. A project (and optionally a solution file) must be in this directory or one of the parent directories to validate and build this code.
> Error: 😵 Project missing. A project (and optionally a solution file) must be in this directory or one of the parent directories to validate and build this code.
> Error: Process completed with exit code 1.
> ```
> 
> The example files already existed before. Should we do something about this or can we ignore it?

@carlossanlop You'll need to add a simple project file to the folder, similar to https://github.com/dotnet/dotnet-api-docs/blob/main/snippets/System.Text/Rune/csharp/RuneSamples.csproj. This is a new requirement to ensure that code samples build correctly. It's already caught some bugs :)

**Comment by carlossanlop** [comment]
@tannergooding please accept my PR https://github.com/tannergooding/dotnet-api-docs/pull/5

I would've pushed directly to this PR branch, but I don't have permission to edit someone's `main` branch 😞

**Comment by carlossanlop** [comment]
Note: the csproj is targeting `net6.0`. Let's hope it builds. 🤞🏼

File: samples/snippets/csharp/VS_Snippets_CLR_System/system.Math.SinCos/CS/sincos.cs [hunk 2]
```diff
@@ -30,6 +31,32 @@ public static void Main()
...
         UseTwoAngles(15.0, 30.0);
         UseTwoAngles(30.0, 45.0);
+
+        Console.WriteLine(
+            "\nWhen you have calls to sin(X) and cos(X) they \n" +
+            "can be replaced with a single call to sincos(x):" );
+
+        UseCombinedSinCosine(15.0);
+        UseCombinedSinCosine(30.0);
+        UseCombinedSinCosine(45.0);
```
Comment (carlossanlop) [inline samples/snippets/csharp/VS_Snippets_CLR_System/system.Math.SinCos/CS/sincos.cs:41]: typo

```suggestion
        UseCombinedSineCosine(15.0);
        UseCombinedSineCosine(30.0);
        UseCombinedSineCosine(45.0);
```

**Comment by carlossanlop** [comment]
> It's already caught some bugs :)

We can confirm the feature works 😄  @gewarren

**Pull request merged**